### PR TITLE
Varia styles for search block

### LINKED ISSUE #1936: Varia: Support Search block better
Search block needs some love in Varia and the child themes.

It looks unstyled in the editor.

Screenshot 2020-04-22 at 16 59 37

When the block is set to centre-aligned, only the label becomes centre-aligned and everything stays the same in the editor.

Screenshot 2020-04-22 at 17 02 39

One the front of the site, the block is centred, but the position of the label is odd.

Screenshot 2020-04-22 at 17 03 58

cc @ianstewart @alaczek

### LINKED ISSUE #2560: Varia: New gutenberg options for search bar are broken on frontend
## Steps to replicate

* Install Gutenberg 9.1
* Insert a Search Bar block
* Choose "toggle search label"
* Change button position to inside 
* Click "Use button with icon"

## Result

Editor:

Screenshot 2020-10-06 at 19 56 41

Frontend:

Screenshot 2020-10-06 at 19 53 32

## Expected

I expected the frontend to look like the editor

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

The search block was introduced pretty recently and Varia doesn't have styles to support it. This PR introduces styles for Varia + Children and color annotation rules so that the styles can be affected by the color customizations by the users.

This is a WIP, since I need to handle customizations for each theme. These are done so far:

**Varia:**

varia

**Alves**

Screenshot 2021-06-01 at 14 21 13

**Balasana:**

balasana

**Barnsbury:**

Screenshot 2021-06-01 at 14 42 01

**Brompton:**

Screenshot 2021-06-01 at 15 52 22

**Coutoire:**

Screenshot 2021-06-02 at 15 24 35

**Dalston:**

Screenshot 2021-06-02 at 15 45 54

**Hever:**

Screenshot 2021-06-02 at 16 06 21

**Leven:**

Screenshot 2021-06-02 at 16 11 42

**Mayland:**

Screenshot 2021-06-02 at 16 19 18

**Maywood:**

Screenshot 2021-06-02 at 16 34 33

**Morden**
Screenshot 2021-06-09 at 12 03 19

**Redhill:**
Screenshot 2021-06-08 at 16 36 41

**Rivington:**
Screenshot 2021-06-09 at 11 24 17

**Rockfield:**
Screenshot 2021-06-09 at 11 20 48

**Shawburn:**
Screenshot 2021-06-09 at 11 37 38

**Stow:**
Screenshot 2021-06-09 at 11 45 37

**Stratford:**
Screenshot 2021-06-09 at 11 57 41

#### Related issue(s):

Closes https://github.com/Automattic/themes/issues/1936

Closes https://github.com/Automattic/themes/issues/2560

### CODE REVIEW

**Comment by MaggieCabrera** [comment]
@kjellr @beafialho @melchoyce I'm still working through these but I think the Alves one is going to need design eyes, I was thinking maybe the inputs should have the same border-radius as the buttons and have the ones with the icon be complete circles. What do you think?

**Comment by kjellr** [comment]
> I was thinking maybe the inputs should have the same border-radius as the buttons and have the ones with the icon be complete circles. What do you think?

That sounds like a good idea on my end. 👍

**Comment by MaggieCabrera** [comment]
@kjellr does this look good?

Screenshot 2021-06-01 at 16 28 49

**Comment by kjellr** [comment]
LGTM! Thanks

File: alves/sass/_extra-child-theme.scss [hunk 1]
```diff
@@ -575,3 +575,22 @@ body:not(.fse-enabled) {
 
 // Updates the Mobile Navigation to be next to the site title.
 @import "../../varia/sass/components/header/_site-mobile-nav-side";
+
+/**
+ * Search block
```
Comment (scruffian) [inline alves/sass/_extra-child-theme.scss:580]: I wonder if we should just add this to Varia?

Comment (scruffian) [reply alves/sass/_extra-child-theme.scss:580]: Actually I don't think that makes sense.

**Comment by scruffian** [comment]
This is now ready for a review. We need to bump the version numbers before we merge.

**Review by danieldudzic — APPROVED** [review]
That was a long one!

Nice job, @MaggieCabrera - LGTM.

The only minor thing that's unnecessary is the redundant padding declaration (`padding: 16px 16px`) here and there, but that's so trivial, I believe this is good to go.

**Pull request merged**